packages/desktop-client/src/components/filters/FiltersMenu.js

```diff
@@ -205,62 +211,96 @@ function ConfigureField({
           {field === 'saved' && 'Existing filters will be cleared'}
         </View>
 
-        <Stack
-          direction="row"
-          align="flex-start"
-          spacing={1}
-          style={{ flexWrap: 'wrap' }}
-        >
-          {type === 'boolean'
-            ? [
-                <OpButton
-                  key="true"
-                  op="true"
-                  selected={value === true}
-                  onClick={() => {
-                    dispatch({ type: 'set-op', op: 'is' });
-                    dispatch({ type: 'set-value', value: true });
-                  }}
-                />,
-                <OpButton
-                  key="false"
-                  op="false"
-                  selected={value === false}
-                  onClick={() => {
-                    dispatch({ type: 'set-op', op: 'is' });
-                    dispatch({ type: 'set-value', value: false });
-                  }}
-                />,
-              ]
-            : ops.map(currOp => (
+        {type === 'boolean' ? (
+          <Stack
+            direction="row"
+            align="flex-start"
+            spacing={1}
+            style={{ flexWrap: 'wrap' }}
+          >
+            <OpButton
+              key="true"
```

Comment:
Since this is no longer in an array, the `key` props are not needed.

Reply:
I switched it back to an array, so leaving the key